numpy/lib/tests/test_nanfunctions.py

```diff
@@ -388,6 +410,94 @@ def test_empty(self):
             assert_equal(res, tgt)
 
 
+class TestNanFunctions_CumSumProd(TestCase, SharedNanFunctionsTestsMixin):
+
+    nanfuncs = [np.nancumsum, np.nancumprod]
+    stdfuncs = [np.cumsum, np.cumprod]
+
+    def test_allnans(self):
+        for f, tgt_value in zip(self.nanfuncs, [0, 1]):
+            with assert_no_warnings():
+                res = f([np.nan]*3, axis=None)
```

Comment:
`pylint` didn't seem to find these and the closest reference I could find was https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements.  Is there a better one?